README.md

```diff
@@ -7,36 +7,32 @@
 
 Standard, Specification, Schema
 
-### Gitter
+### Community Chat
 
-Everyone working on the early stages of the project should join our gitter channel [gitter.im/jsonresume/public](https://gitter.im/jsonresume/public).
+* [Gitter](https://gitter.im/jsonresume/public)
 
-### Getting started
+### Installation
 
+Resume-schema is intended to be installed as a dependency of your project:
+
+```sh
+npm add resume-schema
```

Comment:
`add` isn't a npm command. `--save` is implicitly applied now so it can be removed.

Reply:
done.